content/3.Joining Corda Network/5.Offboarding Workflow.md

```diff
@@ -0,0 +1,22 @@
+If you foresee that one or some of your Corda Network network certificate(s) will not be used permanently or will not re-join in the long-run, you may submit a request to revoke the certificate.
+If you do not intend for the network certificate to be dormant permanently but want to hide the node from the network map, you may shut down the node and it will be hidden from the network map, and once you restart the node it will show up on the network map automatically. You can read more about this feature *(**eventHorizon**)* on the [Corda Documentation Site](https://docs.corda.net/docs/corda-enterprise/4.8/network/network-reqs.html#corda-network-parameters).
+
+If you wish to permanently leave Corda Network, you can terminate your Corda Network contract. All your network certificates will be automatically revoked. Please speak to your R3 Account Manager if that is the case.
```

Comment:
and ensure you have retrieved any assets and data from your node(s) beforehand. [would be good to add something like this here - maybe good to check with someone from Support that this makes sense to say]

Reply:
updated